.claude/scripts/create_polarion_tc.py

```
@@ -0,0 +1,277 @@
+    print()
+
+    if args.dry_run:
+        print("✓ DRY RUN: Draft is valid and ready for creation", file=sys.stderr)
+        sys.exit(0)
+
+    # Load environment
+    env = load_env()
+    polarion_url = env.get('POLARION_URL', 'https://polarion.engineering.redhat.com')
+    polarion_token = env.get('POLARION_TOKEN', '')
+
+    if not polarion_token or polarion_token == 'your_personal_access_token_here':
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "polarion_token"

(S105)